docs/advanced/addons/managed-dhcp.md

```diff
@@ -0,0 +1,248 @@
+---
+sidebar_position: 6
+sidebar_label: Managed DHCP
+title: "Managed DHCP"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/advanced/managed-dhcp"/>
+</head>
+
+_Available as of v1.3.0_
+
+Beginning with v1.3.0, users can configure IP pool information and serve IP addresses to virtual machines running on Harvester clusters via DHCP.
+
+This is accomplished using the vm-dhcp-controller add-on.
+
+## Install and Enable The vm-dhcp-controller Add-on
+
+The vm-dhcp-controller add-on is not packaged with Harvester, but it can be found in the [expreimental-addon repo](https://github.com/harvester/experimental-addons). Users need to install the add-on first:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/harvester/experimental-addons/main/harvester-vm-dhcp-controller/harvester-vm-dhcp-controller.yaml
+```
+
+After the add-on is installed, enable the add-on on the dashboard.
+
+![](/img/v1.3/vm-dhcp-controller/enable-addon.png)
+
+## vm-dhcp-controller
+
+After enabling the vm-dhcp-controller add-on, there will be three kinds of Pods running:
+
+- Controller: Controller reconciles CRD objects to decide what IP addresses are allocated and mapped to what MAC addresses.
+- Webhook: Webhook validates and mutates CRD objects when receiving create/update/delete requests
+- Agent: Agents are spawned on-demand every time users create new IP pool objects. They are responsible for ensuring their internal DHCP lease store is up-to-date by syncing the IP pool objects associated with and serving DHCP requests, i.e., issuing IP addresses according to the MAC -IP mappings recorded in the internal DHCP lease store.
+
+### API
+
+The [vm-dhcp-controller](https://github.com/harvester/vm-dhcp-controller) introduces two new CRDs.
+
+- IPPool (ippl)
+- VirtualMachineNetworkConfig (vmnetcfg)
```

Comment:
I rewrote it because the agent part is a bit unclear. I also added more description to the controller so users can get a sense of how the controller and agents collaborate. PTAL, thanks.